src/components/PageFooter.js

```diff
@@ -0,0 +1,9 @@
+const PageFooter = (props) => {
+    return (
+        <footer>
+            <p>Created at <a href="https://junocollege.com/">Juno College</a></p>
+        </footer>
+    )
+}
+
```

Comment:
great idea to have components for your header and footer, would be really cool to keep these across your app in the future if you wanted to implement routing!